### ISSUE TITLE
Support Accessibility in renderPosts

### ISSUE THREAD

**Issue by RandomJo** [verbb/social-feeds#16]
### What are you trying to do?

There are a few accessibility issues when using the built-in HTML and CSS for `renderPosts`. A few quick modifications could make this better for everyone!

### What's your proposed solution?

It would be great to have the following accessibility issues corrected in `renderPosts`:

- The gray for the post date/time has poor contrast. It would be great to meet WCAG 2.1 AA standards of 4.5:1.
- The link to the post via the icon is missing discernible link text. Something like "View this post on [Source Type]" would be great.

**Comment by engram-design** [comment]
Updated for the next release. To get this early, run `composer require verbb/social-feeds:"dev-craft-5 as 2.0.4"`.

**Comment by engram-design** [comment]
Updated in [2.0.5](https://github.com/verbb/social-feeds/releases/tag/2.0.5)
